keepercommander/commands/record.py

```diff
@@ -209,6 +209,8 @@ def execute(self, params, **kwargs):
         if not force:
             if not title:
                 title = input('...' + 'Title: '.rjust(16))
+                if len(title) == 0:
```

Comment:
I don't think this change is enough.
The empty title can be created when --force parameter is passed.
I can see recordv3.py that has RecordAddCommand.